opencti-platform/opencti-graphql/tests/01-unit/domain/filterKeysSchema-test.ts

```diff
@@ -229,7 +229,7 @@ describe('Filter keys schema generation testing', async () => {
 
     // Stix Core Relationships
     filterDefinition = filterKeysSchema.get(ABSTRACT_STIX_CORE_RELATIONSHIP)?.get(RELATION_FROM_FILTER);
-    expect(filterDefinition?.subEntityTypes.length).toEqual(53); // 51 stix core relationship types + abstract type 'stix-core-relationships'
+    expect(filterDefinition?.subEntityTypes.length).toEqual(56); // 51 stix core relationship types + abstract type 'stix-core-relationships'
```

Comment:
I think the commentary hasn't been updated lately, it's maybe :
```suggestion
    expect(filterDefinition?.subEntityTypes.length).toEqual(56); // 55 stix core relationship types + abstract type 'stix-core-relationships'
```